app/controllers/authenticate_controller.rb

```diff
@@ -47,9 +57,9 @@ def update_config
     handle_authentication_error(e)
   end
 
-  def status_input
+  def status_input(authenticator_name = params[:authenticator])
```

Comment:
Works fine with other authnticators?

Reply:
yes, i just added the option to set the authenticator name manually, but as you can see, the default option is still from the params